.github/actions/PRConventionalCommits-1.3.0/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: 'Conventional Commit In Pull Requests'
+description: 'Check that the PR title contains a valid task type and (optionally) a task number, and label the PR based on the task type'
+inputs:
+
+  task_types:
+    description: 'An array of task types. Example: ["feat","fix","docs","test","ci","refactor","perf","chore","revert"]'
+    required: true
+  ticket_key_regex:
+    description: 'Regular expression to match issue number in PR title, by default does not validate. Example ^PROJECT-\d{2,5}$'
```

Comment:
Do we need this input? Because we don't generally have a ticket pattern

Reply:
Unless its just the standard thing that comes with the lib...

Reply:
That's something from the action's repository. There's an option to check for project/ticket pattern but for the config we put, we won't be using that option